ccc/github.py

```diff
@@ -63,6 +67,31 @@ def github_api_ctor(
         raise ValueError('failed to parse url: ' + str(github_url))
 
     session = github3.session.GitHubSession()
+    original_request = session.request
+
+    def intercepted_request(*args, **kwargs):
+        req = original_request(*args, **kwargs)
+
+        try:
+            method, url = args
+            es_client.store_document(
+                index='github_request',
+                body={
+                    'method': method,
+                    'url': url,
+                    'data': kwargs,
+                    'creation_date': datetime.datetime.now().isoformat(),
+                },
+            )
+        except:
+            # dont raise due to logging
```

Comment:
nit: it is don't (conjunction form for do not)
also: It should already be obvious from try-except w/ warning as single instruction that we do not want to raise. so I would argue the comment could simply be dropped

however, if keeping it, I would rather - as always - see comments that explain WHAT and WHY, rather than repeating the code. we _ignore_ errors from logging. this is different from the comment above (which just states "due to logging").

as a general recommendation, just consider omitting comments in cases where there is no need to explain the underlying idea. but if adding comments, please always be sure to not just repeat the code, and focus on the "higherlevel" semantics, as understood by / relevant for a user | maintainer of the documented piece of code